ee/hogai/summarizer/nodes.py

```diff
@@ -100,5 +104,6 @@ def _construct_messages(self, state: AssistantState) -> list[tuple[str, str]]:
             elif isinstance(message, AssistantMessage):
                 conversation.append(("assistant", message.content))
 
-        conversation.append(("human", SUMMARIZER_INSTRUCTION_PROMPT))
+        conversation.append(("system", SUMMARIZER_RESULTS_PROMPT))
```

Comment:
Two system prompts? It's better to keep a single system prompt.

Reply:
Was playing with a different approach, but actually better to consolidated. Pushed